packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -112,7 +120,7 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
               onClick={this._onToggleSelectGroupClick}
               {...selectAllButtonProps}
             >
-              <Check checked={currentlySelected} />
+              {onRenderCheckbox({ checked: currentlySelected, theme }, onRenderCheckbox)}
```

Comment:
I'm concerned this will have negative performance impact since we're invoking a function in the render path for what was a "static" JSX element. Have we evaluated the performance before and after these changes?

Reply:
To be consistent I adapted the code from DeailsRow.base.tsx which has the same mechanism:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.base.tsx#L274
The overhead would be one function call per header row to be rendered, I doubt the performance difference will be measurable for real word scenarios. Do you have a suggestion on how to improve this?

Reply:
I am fairly certain this wouldn't have a measurable effect. We aren't hitting a hotspot here. We may be adding at most 100 extra function calls, hypothecticall. It will be on the order of nanoseconds of overhead. There is no extra JSX or styling calculations resulting from this.